src/applications/static-pages/static-pages-entry.js

```diff
@@ -200,6 +201,7 @@ createScheduleViewVAAppointmentsPage(
   widgetTypes.SCHEDULE_VIEW_VA_APPOINTMENTS_PAGE,
 );
 createSecureMessagingPage(store, widgetTypes.SECURE_MESSAGING_PAGE);
+createSituationUpdatesBanner(store);
```

Comment:
Does this need its widget types values or not? The function doesn't seem to need it.

Reply:
Ahhh ya, I built this in a way that does not need it with the createSituationsUpdatesBanner grabbing the widgetType directly. Given this goes against the standard pattern here I'm going to make a quick adjustment to make this lineup better with existing patterns.
Thank you for the question!